hedera-node/hedera-app/src/main/java/com/hedera/node/app/fees/MonoGetTopicInfoUsage.java

```diff
@@ -75,16 +76,15 @@ public FeeData computeUsage(final Query query, final ReadableTopicStore topicSto
      * @param topic the PBJ topic
      * @return the Merkle topic
      */
-    @SuppressWarnings("unchecked")
     public static MerkleTopic monoTopicFrom(@NonNull final Topic topic) {
-        final var monoTopic = new MerkleTopic(
+        final MerkleTopic monoTopic = new MerkleTopic(
                 topic.memo(),
-                (JKey) fromPbjKey(topic.adminKey()).orElse(null),
-                (JKey) fromPbjKey(topic.submitKey()).orElse(null),
+                (JKey) fromPbjKey(topic.adminKeyOrElse(Key.DEFAULT)).orElse(null),
+                (JKey) fromPbjKey(topic.submitKeyOrElse(Key.DEFAULT)).orElse(null),
                 topic.autoRenewPeriod(),
                 new EntityId(0, 0, topic.autoRenewAccountNumber()),
                 new RichInstant(topic.expiry(), 0));
-        monoTopic.setRunningHash(unwrapPbj(topic.runningHash()));
+        monoTopic.setRunningHash(PbjConverter.asBytes(topic.runningHash()));
```

Comment:
We should probably delete `unwrapPbj` if its not being used anywhere

Reply:
Agreed, removed